### PR TITLE
go.mod: replace inet.af/tcpproxy -> github.com/inetaf/tcpproxy

### PR DESCRIPTION
**Pull request by AkihiroSuda**
Changes: https://github.com/inetaf/tcpproxy/compare/91f861402626c6ba93eaa57ee257109c4f07bd00...3ce58045626c8bc343a593c90354975e61b1817a

https://github.com/inetaf/tcpproxy/commit/3ce58045626c8bc343a593c90354975e61b1817a says "change import path after Taliban seized it or something":
- https://github.com/inetaf/tcpproxy/issues/39

It is *unlikely* that `inet.af/tcpproxy v0.0.0-20221017015627-91f861402626` cached in proxy.golang.org is compromised.

### CODE REVIEW

**Review by jandubois — APPROVED** [review]
Thanks, LGTM

File: go.mod [hunk 1]
```diff
@@ -127,6 +127,7 @@ require (
 	gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7 // indirect
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	gvisor.dev/gvisor v0.0.0-20231023213702-2691a8f9b1cf // indirect
+	inet.af/tcpproxy v0.0.0-20221017015627-91f861402626 // indirect
```
Comment (AkihiroSuda) [inline go.mod:130]: - https://github.com/containers/gvisor-tap-vsock/pull/399

Comment (jandubois) [reply go.mod:130]: We could redirect the module in our `go.mod` file until `gvisor-tap-vsock` has a new release.

Comment (AkihiroSuda) [reply go.mod:130]: `replace inet.af/tcpproxy => github.com/inetaf/tcpproxy v0.0.0-20240214030015-3ce
58045626c` doesn't seem to work?

```
go: github.com/inetaf/tcpproxy@v0.0.0-20240214030015-3ce58045626c used for two different module paths (github.com/inetaf/tcpproxy and inet.af/tcpproxy)
```

**Review by jandubois — APPROVED** [review]

**Pull request merged**